### PR TITLE
HPCC-15461 FTSlave incorrectly reports/logs chunk parameters at start of the transfer.

### PR DESCRIPTION
**Pull request by AttilaVamos**
Fix and improve progress report/log info

Signed-off-by: Attila Vamos attila.vamos@gmail.com

### CODE REVIEW

**Comment by AttilaVamos** [comment]
@jakesmith  please review.

File: dali/ft/ftbase.cpp [hunk 1]
```diff
@@ -573,7 +573,7 @@ MemoryBuffer & OutputProgress::deserializeExtra(MemoryBuffer & in, unsigned vers
 static const char * const statusText[] = {"Init","Active","Copied","Renamed"};
 void OutputProgress::trace()
 {
-    LOG(MCdebugInfoDetail, unknownJob, "[%d] %s  %" I64F "d[%x]->%" I64F "d[%x]", whichPartition, statusText[status], inputLength, inputCRC, outputLength, outputCRC);
+    LOG(MCdebugInfoDetail, unknownJob, "Chunk%d status: %s  input length: %" I64F "d[CRC:%x] -> output length:%" I64F "d[CRC:%x]", whichPartition, statusText[status], inputLength, inputCRC, outputLength, outputCRC);
```
Comment (jakesmith) [inline dali/ft/ftbase.cpp]: trivial: missing space after 'Chunk'

File: dali/ft/fttransform.cpp [hunk 2]
```diff
@@ -669,7 +669,11 @@ void TransferServer::transferChunk(unsigned chunkIndex)
     PartitionPoint & curPartition = partition.item(chunkIndex);
     OutputProgress & curProgress = progress.item(chunkIndex);
 
-    LOG(MCdebugProgress, unknownJob, "Begin to transfer chunk %d: Start at length %" I64F "d", chunkIndex, curProgress.inputLength);
+    StringBuffer targetPath;
+    curPartition.outputName.getPath(targetPath);
+    const char * ptargetPath = targetPath.str();
```
Comment (jakesmith) [inline dali/ft/fttransform.cpp]: trivial: don't really need this line, use targetPath.str() in line below.

File: dali/ft/fttransform.cpp [hunk 3]
```diff
@@ -669,7 +669,11 @@ void TransferServer::transferChunk(unsigned chunkIndex)
...
+    LOG(MCdebugProgress, unknownJob, "Begin to transfer chunk %d (offset %" I64F "d, size:%" I64F "d) to target:'%s' (offset: %" I64F "d, size: %" I64F "d) ",
```
Comment (jakesmith) [inline dali/ft/fttransform.cpp]: trivial: inconsistent colons/spaces, i.e. 1st 'offset' has no colon after, 1st size: has no space after.

**Comment by jakesmith** [comment]
@AttilaVamos - a few trivial things, but otherwise fine.

**Comment by AttilaVamos** [comment]
@jakesmith updated.

File: dali/ft/fttransform.cpp [hunk 4]
```diff
@@ -669,7 +669,10 @@ void TransferServer::transferChunk(unsigned chunkIndex)
     PartitionPoint & curPartition = partition.item(chunkIndex);
     OutputProgress & curProgress = progress.item(chunkIndex);
 
-    LOG(MCdebugProgress, unknownJob, "Begin to transfer chunk %d: Start at length %" I64F "d", chunkIndex, curProgress.inputLength);
+    StringBuffer targetPath;
+    curPartition.outputName.getPath(targetPath);
+    LOG(MCdebugProgress, unknownJob, "Begin to transfer chunk %d (offset: %" I64F "d, size: %" I64F "d) to target:'%s' (offset: %" I64F "d, size: %" I64F "d) ",
```
Comment (jakesmith) [inline dali/ft/fttransform.cpp]: trivial: not new, but these should be %u 's really.

**Comment by jakesmith** [comment]
Good to merge.

@richardkchapman

**Pull request merged**